### DISCUSSION TITLE
Zepp OS v3.0 release info

### DISCUSSION THREAD

**Discussion by ismaelestalayo** [zepp-health/zeppos-forum#250] [Developer Feedback]
I just saw the newly announced Amazfit Balance that comes with ZeppOS 3.0, yet there's no info about what other devices that version will support (GTR4, Cheetah...) and when it will be launched as an OTA update (beta versions available?).

Can you provide some clarification on that?

**Comment by juanitadion** [comment]
Products are specifically designed to address a range of consumer needs, and it's unrealistic to expect every product to be compatible with every software version. Just as not all computer motherboards can accommodate the latest graphic cards due to hardware limitations, some older devices may not optimally support newer software updates. Each operating system is intricately developed for certain hardware configurations. We will inform our users when older devices qualify for upgrades. Promising universal compatibility would be misleading.

**Reply by ismaelestalayo** [reply]
Yes, I understand that not all devices can be compatible with every future release ever. All hardware has its limitations and its software support life cycle.

However I was just wondering if there was any official information of what current devices will get support for ZeppOS 3.0, because in the [documentation](https://docs.zepp.com/docs/reference/related-resources/device-list/) you only list the newly released Balanced as supported, while there's no info for the rest.

I ask this as a developer of watchfaces and apps for ZeppOS (and owner of a GTR4), because in my opinion a device not getting even a single year of OS updates support does not sit well with users, who usually expect a minimum support for their device.

**Comment by juanitadion** [comment]
@ismaelestalayo Different products are designed to cater to varied consumer needs, and only some products are compatible with all software versions. Unlike computers, where not every motherboard can support the latest graphic cards, devices have firmware limitations. Some older devices might not be able to support newer software updates at their best performance. Each operating system is uniquely tailored to specific hardware. In summary, when our previous devices are eligible for an upgrade, we will announce it. It's not feasible for every product to support every software version, and it would be misleading to suggest otherwise.
